http/1.8/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -0,0 +1,696 @@
+. Set *Host* to `localhost` and *Port* to `8081`:
+
+image::http-request-1.png[HTTP Request global configuration window with Host field set to localhost and Port field to 8081]
+[start=5]
+. Set *TLS configuration* to *Edit inline*.
+. In the *Trust Store Configuration* section, set *Path* to `keystore.jks` and *Password* to `Mypassword`:
+
+image::http-request-2.png[Trust Store Configuration window with Path field set to keystore.jks and Password field to Mypassword]
+
+[start=7]
+. Set *Authentication* to *Basic authentication*.
+. Set *Username* to `testuser` and *Password* to `Mypassword`.
+
+image::http-request-3.png[Authentication field set to Basic authentication and User field set to testuser and Password field to Mypassword]
+
+[start=9]
+. Click *OK*.
+. In the HTTP *Request* configuration screen, set *Path* to `/testpath`:
+
+In the *Configuration XML* editor, the `host`, `port`, `path`, `trust-store`, and `authentication` configurations look like this example:
+
+[source,xml,linenums]
+----
+<http:request-config name="HTTP_Request_configuration">
+		<http:request-connection host="localhost" port="8081">
+			<tls:context >
+				<tls:trust-store path="keystore.jks" password="Mypassword" />
+			</tls:context>
+			<http:authentication >
+				<http:basic-authentication username="testuser" password="Mypassword" />
+			</http:authentication>
+		</http:request-connection>
+	</http:request-config>
+  <flow name="httplistenerresponseFlow1" >
+  <http:request method="GET" config-ref="HTTP_Request_configuration" path="/testpath"/>
+</flow>
+----
+
+== Configure HTTP Request Connectivity Testing
+
+The HTTP *Request* operation requires a global request configuration with a connection provider. Since HTTP Connector version 1.7.0, the connection provider supports connectivity testing.
+
+=== Configure Minimal Connectivity Testing
+
+To configure connectivity testing in the request global configuration without depending on the *Request* operation, add the `<http:connectivity-test>` and `requestPath` parameters in the `<http:request-connection>` tag as follows:
+
+[source,xml,linenums]
+----
+<http:request-config name="requestConfigWithOnlyPath">
+  <http:request-connection host="localhost" port="80" >
+    <http:connectivity-test requestPath="/thePath" />
+  </http:request-connection>
+</http:request-config>
+----
+
+After you configure the connectivity testing, you can test the connection in Studio:
+
+. Navigate to the *Global Elements* tab.
+. Select *HTTP Request configuration (Configuration)*.
+. Click *Edit* to open the global configuration window.
+. Click *Test connection...* to trigger a request to the URL `+http://localhost:80/thePath+`.
+
+If a connection is established, a request is sent, and a response is received. The test connection is successful if the response status code is in the range of 0-399. Otherwise, the test connection failed.
```

Comment:
```suggestion
If a connection is established, the connector sends a request and then receives a response. The test connection is successful if the response status code is in the range `0-399`. Otherwise, the test connection failed.
```